api/GPU.json

```
@@ -32,11 +32,29 @@
             }
           ],
           "edge": "mirror",
-          "firefox": {
-            "version_added": "141",
-            "partial_implementation": true,
-            "notes": "Supported on Windows only, in all contexts except for service workers."
-          },
+          "firefox": [
+            {
+              "version_added": "147",
+              "impl_url": [
+                "https://bugzil.la/2004105",
+                "https://bugzil.la/1942431"
+              ],
+              "partial_implementation": true,
+              "notes": "Supported on Windows and macOS on Apple silicon, in all contexts except for service workers."
```

Re https://github.com/mdn/browser-compat-data/pull/28604#issuecomment-3633395422

> We discussed this in today's BCD meeting, and it's a bit tricky:
> 
> * The DOM interfaces seem to actually be available on all platforms, even if `requestAdapter()` will not return an adapter.
> * The notes currently don't explain what "unsupported" means.

@caugner Thank you. Here is some proposed text then. If you are OK with this I'll roll it out everywhere. If not, can you please counter-propose some text?


```suggestion
              "notes": "Supported on Windows and macOS on Apple silicon, in all contexts except for service workers. On other platforms the interfaces are exposed but are not accessible (`GPU.requestAdapter()` returns `null`)."
```